src/main/java/org/opensrp/web/rest/LocationResource.java

```diff
@@ -68,10 +67,11 @@ public void setLocationService(PhysicalLocationService locationService) {
 
 	@RequestMapping(value = "/{id}", method = RequestMethod.GET, produces = { MediaType.APPLICATION_JSON_VALUE })
 	public ResponseEntity<String> getByUniqueId(@PathVariable("id") String id,
-			@RequestParam(value = IS_JURISDICTION, defaultValue = FALSE, required = false) boolean isJurisdiction) {
+			@RequestParam(value = IS_JURISDICTION, defaultValue = FALSE, required = false) boolean isJurisdiction,
+			@RequestParam(value = RETURN_GEOMETRY, defaultValue = FALSE, required = false) boolean returnGeometry) {
```

Comment:
For this since its a single location we are returning default RETURN_GEOMETRY to `true`